addon/HomepageStudy.jsm

```diff
@@ -71,19 +76,77 @@ function getHomepage() {
   return eTLD;
 }
 
+/**
+ * Returns the correct parameters depending if it's a standalone execution or a simulation.
+ * @param {boolean} isSimulation - Boolean indicating if the addon is run for a simulation.
+ * @param {string} rapporPath  - Path of the RAPPOR simulator.
+ */
+function getParams(isSimulation, rapporPath, instance) {
```

Comment:
Should this live in the simulator module?

Reply:
Done